packages/@govuk-frontend/date-input/_date-input.scss

```diff
@@ -21,6 +21,10 @@
     padding-bottom: 2px;
   }
 
+  .govuk-c-date-input__input {
```

Comment:
Is this change meant to be here?

Reply:
it's all latest changes i guess

Reply:
That's not from the latest round of changes: https://github.com/alphagov/govuk-frontend/commit/b48d3913a4b8f41fda60a0afa22641a3a54e99dd#diff-e94394b2ac1d4f73991af98e4fa34fa3